sentry_sdk/integrations/starlette.py

```diff
@@ -389,8 +384,9 @@ def _sentry_request_response(func):
 
             async def _sentry_async_func(*args, **kwargs):
                 # type: (*Any, **Any) -> Any
-                hub = Hub.current
-                integration = hub.get_integration(StarletteIntegration)
+                integration = sentry_sdk.get_client().get_integration(
+                    StarletteIntegration
+                )
                 if integration is None:
                     return await old_func(*args, **kwargs)
```

Comment:
We could used `ensure_integration_enabled_async` here

Reply:
See above. Will open an issue for this.

Reply:
see answer above :-)